bundler/spec/commands/console_spec.rb

```diff
@@ -2,6 +2,12 @@
 
 RSpec.describe "bundle console", readline: true do
   before :each do
+    begin
+      require "irb"
```

Comment:
Should we add `irb` to dev gems instead (any of https://github.com/rubygems/rubygems/tree/30df09bb8175cc6e49766374bd1ef7fc42355d06/tool/bundler) to ensure it is present for bundler test suite?

Reply:
I was going to suggest the same thing. Otherwise this will become dead code eventually, and `bundle console` no longer tested.

Also, I was trying to fix the same thing at #8450, but did not get that working.